multicluster/Manifest/gslb-controller.yaml

```diff
@@ -1,12 +1,15 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: v1
+kind: Pod
 metadata:
   name: gslb-k8s-ingress-controller
+  namespace: default
+  labels:
+    app: gslb-k8s-ingress-controller
```

Comment:
Please change the initial lines to as mentioned below

apiVersion: apps/v1
kind: Deployment
metadata:
  name: gslb-k8s-ingress-controller
spec:
  selector:
    matchLabels:
      app: gslb-k8s-ingress-controller
  replicas: 1
  template:
    metadata:
      name: gslb-k8s-ingress-controller
      labels:
        app: gslb-k8s-ingress-controller
      #annotations:
    spec:
      serviceAccountName: gslb-k8s-role
      containers:
      - name: gslb-k8s-ingress-controller